src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -46,9 +48,35 @@ const PackageDetailBreadcrumbs = ({ cosmosPackage }) => {
 const METHODS_TO_BIND = [
   "handleInstallModalClose",
   "handleConfigureInstallModalOpen",
-  "handleInstallModalOpen"
+  "handleInstallModalOpen",
+  "handlePackageVersionChange"
 ];
 
+const formatPackageVersions = selectedPackage => {
+  if (selectedPackage.packageVersions == null) {
+    return [];
+  }
+  const versions = selectedPackage.packageVersions;
+
+  return Object.keys(versions)
+    .sort((a, b) => {
+      if (+versions[a] > +versions[b]) {
+        return 1;
+      }
+      if (+versions[a] < +versions[b]) {
+        return -1;
+      }
+
```

Comment:
I think it would make more sense to reverse these
```
      if (+versions[a] > +versions[b]) {
        return -1;
      }
      if (+versions[a] < +versions[b]) {
        return 1;
      }
```
I'm used to previewing versions from latest to oldest. I think it makes more sense logically as the user will probably look for the latest version and it is more standard to promote and bubble up the latest.

![image](https://user-images.githubusercontent.com/180432/30074501-9eeeec5c-9226-11e7-9e5a-848630a48745.png)

Reply:
Agree, let me just confirm with design team